test_3_add_up_numbers.py

```diff
@@ -2,8 +2,7 @@
 
 def add_up_numbers(arr):
   # Remove the `return None` code below and add your own code.
-  return None
-
+ return sum(int(x) for x in arr)
```

Comment:
Somewhere out on the interwebs, I read that a best practice for use of the python sum function when using "raw input" was to convert it to a number to ensure it works. 

It was either this discussion, or something similar: 
http://stackoverflow.com/questions/11344827/summing-elements-in-a-list

Thx!